packages/devtools_app/lib/src/shared/ui/search.dart

```diff
@@ -919,8 +919,10 @@ class SearchField<T extends SearchControllerMixin> extends StatefulWidget {
   /// triggered.
   final VoidCallback? onClose;
 
-  /// Whether the search field should soft wrap lines, expanding the height.
-  final bool _shouldExpandHeight;
+  /// The maximum number of lines, by default one.
+  ///
+  /// Can be set to null to remove the restriction; must not be zero.
```

Comment:
if it must not be zero, can you add an assert to the constructor?

Reply:
Good call! Done